bin/daemon.php

```diff
@@ -163,9 +163,15 @@
 		exit(1);
 	} elseif ($pid) {
 		// The parent process continues here
+		file_put_contents($pidfile, $pid);
+		if (!is_readable($pidfile)) {
```

Comment:
`file_put_contents` returns the number of bytes written, so this can be shortened.
```suggestion
		if (!file_put_contents($pidfile, $pid)) {
```